includes/collection/class-followers.php

```diff
@@ -310,14 +312,12 @@ public static function get_inboxes( $user_id ) {
 	 * @return array The list of Inboxes.
 	 */
 	public static function get_inboxes_for_activity( $json, $actor_id, $batch_size = 50, $offset = 0 ) {
-		$inboxes = self::get_inboxes( $actor_id );
-
-		if ( self::maybe_add_inboxes_of_blog_user( $json, $actor_id ) ) {
-			$inboxes = \array_fill_keys( $inboxes, 1 );
-			foreach ( self::get_inboxes( Actors::BLOG_USER_ID ) as $inbox ) {
-				$inboxes[ $inbox ] = 1;
-			}
-			$inboxes = \array_keys( $inboxes );
+		$activity = \json_decode( $json, true );
+		// Only if this is an Update or Delete. Create handles its own "Announce" in dual user mode.
+		if ( \in_array( $activity['type'] ?? null, array( 'Update', 'Delete' ), true ) ) {
+			$inboxes = Actors::get_inboxes();
```

Comment:
The way I'm reading this, it would send a post update to all inboxes we have on file, even if they're not following the actor or blog actor. Is that accurate?

Reply:
Should it do that?

I guess I'd understand if we wanted to cast a wide net for actor deletions, but this is much broader.

Reply:
maybe we could limit this to Deletes where the activity -> object -> type is in Actor::TYPES?

Reply:
Yeah, let's limit it to only Deletes for now